generators/app/templates/_package.json

```diff
@@ -108,9 +109,9 @@
     "@types/node": "^8.10.13",
     "codelyzer": "^4.3.0",
 <% if (props.target.includes('electron')) { -%>
-        "electron": "~1.8.2",
-        "electron-packager": "^8.6.0",
-        "electron-winstaller": "^2.2.0",
+    "electron": "^2.0.6",
+    "electron-packager": "^12.1.0",
+    "electron-winstaller": "^2.6.4",
```

Comment:
Is `"electron-winstaller": "^2.6.4"` actually used? Or is it needed by electron packager maybe?